### PR TITLE
Currency Symbols Added

### PR DESCRIPTION
**Pull request by Tarun-Arora**
Currency Symbols Added #5

### CODE REVIEW

File: dictionary.txt [hunk 1]
```diff
@@ -930,3 +930,7 @@ _rp	₎
 \daleth	ℸ	
 ^epsilon	ᵋ	
 \rightangle	∟	
+\rupee	₹	
+\dollar	$	
+\euro	€	
+\japyen	¥
```
Comment (DenverCoder1) [inline dictionary.txt:936]:
```suggestion
\yen	¥	
```

**Review by DenverCoder1 — COMMENTED** [review]
Looks great! Could be very useful. Maybe remove jap part from japyen?

File: dictionary.txt (hunk 1 above)
Comment (Tarun-Arora) [reply dictionary.txt]: Updated This.

**Review by DenverCoder1 — APPROVED** [review]
Thanks for contributing! :tada:

**Pull request merged**